paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -292,6 +307,10 @@ private fun LabelUI(label: String, color: Int?) {
             style = StripeTheme.primaryButtonStyle.getComposeTextStyle(),
             modifier = Modifier
                 .padding(start = 4.dp, end = 4.dp, top = 4.dp, bottom = 5.dp)
+                .semantics {
+                    // Accessibility should be handled by its parent, PrimaryButton.
```

Comment:
I think this is probably worth going a little further. Something along the lines of "This shouldn't be visible for accessibility purposes due to the content description and the click listener being defined outside of compose, in PrimaryButton."